### PR TITLE
create a grid for battleship field

### PR DESCRIPTION
**Pull request by Jakub686**

### CODE REVIEW

File: src/main/java/org/example/model/Grid.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.example.model;
+
+public class Grid {
+    public String[][] getGrid(int row, int column) {
+        String[][] grid = new String[row + 1][column + 1];
+        grid[0][0] = " ";
```
Comment (BartlomiejSzczotka) [inline src/main/java/org/example/model/Grid.java:6]: why you assing here empty value, do it in different way

File: src/main/java/org/example/Ships.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+public class Ships {
+    static int[] ship1Five = new int[11]; // This ship is made of five parts OOOOOO 1 player
+    static int[] ship1Four = new int[9]; // This ship is made of four parts  OOOO 1 player
+    static int[] ship1Three1 = new int[7]; // This ship is made of three parts OOO 1 player
+    static int[] ship1Three2 = new int[7]; // This ship is made of three parts OOO 1 player
+    static int[] ship1Two = new int[5]; // This ship is made of two parts OO 1 player
+    static int[] ship2Five = new int[11]; // This ship is made of five parts OOOOOO 2 player
+    static int[] ship2Four = new int[9]; // This ship is made of four parts  OOOO 2 player
+    static int[] ship2Three1 = new int[7]; // This ship is made of three parts OOO 2 player
+    static int[] ship2Three2 = new int[7]; // This ship is made of three parts OOO 2 player
```
Comment (BartlomiejSzczotka) [inline src/main/java/org/example/Ships.java:12]: three parts 000 hmmm... why ten table lenght is [7]? Plz explain why this table lenght is like that?

**Review by BartlomiejSzczotka — APPROVED** [review]

File: src/main/java/org/example/model/Grid.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package org.example.model;
+
+public class Grid {
+    public String[][] getGrid(int row, int column) {
+        String[][] grid = new String[row + 1][column + 1];
```
Comment (BartlomiejSzczotka) [inline src/main/java/org/example/model/Grid.java:5]: why row +1? why you don't pass just greater value instead? -> put just row and column without +1

**Review by BartlomiejSzczotka — APPROVED** [review]

File: src/main/java/org/example/Ships.java (hunk 2 above)
Comment (Jakub686) [reply src/main/java/org/example/Ships.java:12]: for example, shipfive is made of 5 pieces, every piece has two coordinates X and Y, so 5*2 = 10. Last index in array is for boolean status, which shows is this ship is sinked or not. So Shipfive have 10 coordinates + one more index, that makes ShipFive has array of 11 indexes etc.

File: src/main/java/org/example/model/Grid.java (hunk 3 above)
Comment (Jakub686) [reply src/main/java/org/example/model/Grid.java]: if we want to create a field 10x10 we have to make grid bigger by +1 for description row and description column, grid in assumption is generic, so i couldn't set hardcoded value

image

**Pull request merged**